CODE_OF_CONDUCT.md

```diff
@@ -1,75 +1,48 @@
 # Contributor Covenant Code of Conduct
 
-## Our Pledge
+## 📌 Our Pledge
 
-We as members, contributors, and leaders pledge to make participation in our
-community a harassment-free experience for everyone, regardless of age, body
-size, visible or invisible disability, ethnicity, sex characteristics, gender
-identity and expression, level of experience, education, socio-economic status,
-nationality, personal appearance, race, religion, or sexual identity
-and orientation.
+We, as members, contributors, and leaders, pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
 
-We pledge to act and interact in ways that contribute to an open, welcoming,
-diverse, inclusive, and healthy community.
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community. 👥💙
 
-## Our Standards
+## 📜 Our Standards
 
-Examples of behavior that contributes to a positive environment for our
-community include:
+Examples of behavior that contribute to a positive environment for our community include:
 
-* Demonstrating empathy and kindness toward other people
-* Being respectful of differing opinions, viewpoints, and experiences
-* Giving and gracefully accepting constructive feedback
-* Accepting responsibility and apologizing to those affected by our mistakes,
-  and learning from the experience
-* Focusing on what is best not just for us as individuals, but for the
-  overall community
+* Demonstrating empathy and kindness toward other people. 💖
+* Being respectful of differing opinions, viewpoints, and experiences. 🤝✨
+* Giving and gracefully accepting constructive feedback. 📝✉️
+* Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience. 🙇‍♀️🙇‍♂️
+* Focusing on what is best not just for us as individuals, but for the overall community. 🌟🤝
 
 Examples of unacceptable behavior include:
 
-* The use of sexualized language or imagery, and sexual attention or
-  advances of any kind
-* Trolling, insulting or derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or email
-  address, without their explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
-  professional setting
+* The use of sexualized language or imagery, and sexual attention or advances of any kind. 🚫❌🙅‍♀️
```

Comment:
Remove the emojis added at the end of sentences. It's not looking professional here.

Reply:
Do the same for all sentences.